integration_tests/__init__.py

```diff
@@ -26,14 +26,15 @@
 import fixtures
 import pexpect
 from unittest import mock
+import testscenarios
 import testtools
 from testtools import content
 from testtools.matchers import MatchesRegex
 
 from snapcraft.tests import fixture_setup
 
 
-class TestCase(testtools.TestCase):
+class TestCase(testscenarios.WithScenarios, testtools.TestCase):
```

Comment:
look at the other integration tests using scenarios, this really really isn't needed.